packages/expo-analytics-segment/ios/EXSegment/EXSegment.m

```diff
@@ -132,11 +132,12 @@ @implementation EXSegment
 UM_EXPORT_METHOD_AS(screenWithProperties,
                     screenWithProperties:(NSString *)screenName
                     withProperties:(NSDictionary *)properties
+                    withOptions:(NSDictionary *)options
```

Comment:
I think it would be better if we use `nullable` and `nonnull` keywords so that other people can easily spot what the author had in mind 😉 But it's more like a general comment and not to this specific line as this method just passes things to another method 😄 

```suggestion
                    withOptions:(nullable NSDictionary *)options
```

Reply:
👍 , but please also test that this didn't break anything — historically unimodules didn't support these attributes to method signature, I don't remember if they do now.

Reply:
afaik it was about `__unused` attribute and not `nullable`/`nonnull` — see here https://github.com/expo/universe/issues/2714

Reply:
If so, great, I was under the impression any attribute would break it. 👍

Reply:
good idea, I added this on android but forgot on iOS, so it's good to have them match. and tested in a bare project 👍